package.json

```diff
@@ -80,6 +80,7 @@
 		"react-router": "^5.1.1",
 		"react-router-dom": "^5.1.1",
 		"react-slick": "^0.25.2",
+		"react-table": "^7.0.0-beta.12",
```

Comment:
You should not build your component on a `beta` version of a dependency. Try installing a `stable` version instead

Reply:
We are removing this dependency . Building our own.